openedx/core/djangoapps/programs/utils.py

```diff
@@ -111,6 +115,11 @@ def invert_programs(self):
 
         for program in self.programs:
             for course in program['courses']:
+                course_uuid = course['uuid']
```

Comment:
What is this doing?  Doesn't the next line reassign this variable before it's used?

Reply:
No the next line uses this variable in a comparison

Reply:
Yup!  Definitley read `for course_uuid in` not `if course_uuid in`

Carry on! [Nothing to see here ](https://www.youtube.com/watch?v=5NNOrp_83RU)